cmake/projects/CURL/hunter.cmake

```diff
@@ -27,6 +27,15 @@ hunter_add_version(
 )
 
 
+if (ANDROID OR IOS)
+  set(_curl_cmake_args
```

Comment:
If somebody set this variable outside and platform is not Android/iOS we can carry junk to the option. Just add `set(_curl_cmake_args "")` for `else` condition.